sdk/webpubsub/Azure.Messaging.WebPubSub.Client/src/WebPubSubClient.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Buffers;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Net.WebSockets;
+using System.Runtime.ExceptionServices;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using System.Web;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Messaging.WebPubSub.Clients
+{
+    /// <summary>
+    /// The WebPubSubService PubSub client.
+    /// </summary>
+    [SuppressMessage("Usage", "AZC0007:DO provide a minimal constructor that takes only the parameters required to connect to the service.", Justification = "WebPubSub clients are Websocket based and don't use ClientOptions functionality")]
+    [SuppressMessage("Usage", "AZC0004:DO provide both asynchronous and synchronous variants for all service methods.", Justification = "Synchronous methods doesn't make sense in the scenario of WebPubSub client")]
+    [SuppressMessage("Usage", "AZC0015:Unexpected client method return type.", Justification = "WebPubSubClient is a pure data plane client that don't need to return type as a management client does.")]
+    [SuppressMessage("Design", "CA1001:Types that own disposable fields should be disposable", Justification = "We don't want user to use client within a using block")]
+    public class WebPubSubClient
```

Comment:
This issue has been discussed with architects. The conclusion is we get rid of the interface to avoid the hint of using the client in `using ()` blocks. If the interface is useful, we can add it later without breaking change